crates/bitcoin-da/src/service.rs

```diff
@@ -183,13 +215,16 @@ impl BitcoinService {
             to_batch_proof_prefix: chain_params.to_batch_proof_prefix,
             inscribes_queue: tx,
             tx_backup_dir: tx_backup_dir.to_path_buf(),
+            monitoring,
         })
     }
 
     pub fn spawn_da_queue(
         self: Arc<Self>,
         mut rx: UnboundedReceiver<Option<SenderWithNotifier<TxidWrapper>>>,
     ) {
+        self.monitoring.clone().spawn();
+
         tokio::spawn(async move {
             let mut prev_utxo = match self.get_prev_utxo().await {
                 Ok(Some(prev_utxo)) => Some(prev_utxo),
```

Comment:
should we get the prev_utxo from the monitoring service now?

and update it from there so if tx is bumped, we use an output from the bump tx